lib/gitlab/client/commits.rb

```diff
@@ -0,0 +1,84 @@
+class Gitlab::Client
+  # Defines methods related to repositories.
```

Comment:
Please add `@see https://github.com/gitlabhq/gitlabhq/blob/master/doc/api/commits.md` below it.